charts/refinery/values.yaml

```diff
@@ -230,6 +230,11 @@ config:
     # listen for requests for /metrics.
     MetricsListenAddr: 0.0.0.0:9090
 
+  # When Name is left blank, refinery sets up a ConfigMap with the rules defined below.
+  # The alternative is to pass in your own ConfigMap with the rules defined there.
+  ConfigMap:
```

Comment:
Is there a reason that this needs to be nested 3 levels deep? Could we use something like `ConfigMapName` instead?

Reply:
Not necessarily. I can make that change. :)